parlai/agents/ir_baseline/ir_baseline.py

```diff
@@ -139,20 +147,24 @@ def observe(self, obs):
     def act(self):
         if self.opt.get('datatype', '').startswith('train'):
             self.dictionary.act()
-
+        
         obs = self.observation
         reply = {}
         reply['id'] = self.getID()
         
         # Rank candidates
+        cands = None
         if 'label_candidates' in obs and len(obs['label_candidates']) > 0:
-            # text = obs['text']
+            cands = obs['label_candidates']
+        if hasattr(self, 'label_candidates'):
+            cands = self.label_candidates
```

Comment:
can you add `# override label candidates with candidate file if set` to make it obvious the ordering is intentional?